test/ui/test_lobby.js

```diff
@@ -0,0 +1,113 @@
+import {Lobby} from '../../app/javascript/src/game/ui/lobby.js';
+
+const assert = require('assert');
+const sinon = require('sinon');
+
+const jsdom = require("jsdom");
+const { JSDOM } = jsdom;
+
+describe('Lobby', function() {
+  describe('#update', function() {
+
+    /**
+    * Sets up a new DOM with a 'lobby' div and no players
+    **/
+    beforeEach(function() {
+      const jsdom = new JSDOM('<!DOCTYPE html><html><body><div id="lobby"></div></body></html>');
+      const { window } = jsdom
+      global.window = window;
+      global.document = window.document;
+    });
+
+    /**
+    * Helper function for testing if the DOM body element contains text
+    **/
+    const bodyContains = (text) => {
+      return global.document.body.innerHTML.indexOf(text) > -1;
+    }
+
+    /**
+    * Default payload of 2 players
+    **/
+    const payload = {players: [
```

Comment:
Here it's payload and JSON, but in the object it's an array called players

Reply:
Not sure what you mean here, `payload` is just an object with a `players` property which is an array - the same as would be used to instantiate the `Lobby` class.